docs/examples/opentracing/main.py

```diff
@@ -12,13 +12,16 @@
 trace.set_tracer_provider(TracerProvider())
 tracer_provider = trace.get_tracer_provider()
 
-# Configure the tracer to export traces to Jaeger
-jaeger_exporter = JaegerExporter(
-    agent_host_name="localhost",
-    agent_port=6831,
+# Create an OTLP gRPC span exporter
+otlp_exporter = OTLPSpanExporter(
+    endpoint="localhost:4317",
```

Comment:
After a test on my local with `OTEL_EXPORTER_OTLP_ENDPOINT`, I think this does have to have the `http` in front of it, i.e.

```suggestion
    endpoint="http://localhost:4317",
```

Reply:
@tammy-baylis-swi should i request for review again